### PR TITLE
fix: visual bugs

### PR DESCRIPTION
**Pull request by TGlide**
## What does this PR do?

Fixes some minor visual bugs

## Test Plan

Manual

## Related PRs and Issues

N/A

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

Yes

### CODE REVIEW

File: src/lib/components/empty.svelte [hunk 1]
```diff
@@ -40,12 +41,13 @@
             </button>
             <slot>
                 <div class="u-text-center">
-                    <Heading size="7" tag="h2">Create your first {target} to get started.</Heading>
+                    <Heading size="7" tag="h2" trimmed={false}
+                        >Create your first {target} to get started.</Heading>
```
Comment (ArmanNik) [inline src/lib/components/empty.svelte:45]: Nitpick, can we fix this so the `>` doesn't go to a new line? :laughing:

**Review by ArmanNik — APPROVED** [review]

File: src/lib/components/empty.svelte (hunk 1 above)
Comment (TGlide) [reply src/lib/components/empty.svelte]: Done!

**Comment by TGlide** [comment]
@ArmanNik asked for a re-review because I also added a fix for the footer not being stuck to the bottom on some pages

File: src/lib/layout/shell.svelte [hunk 2]
```diff
@@ -92,4 +92,8 @@
             grid-template-columns: auto 1fr !important;
         }
     }
+
+    .main-content {
+        flex-grow: 1;
+    }
```
Comment (ArmanNik) [inline src/lib/layout/shell.svelte:98]: @TGlide I don't think we need this... Elad should have fixed it and it's merged in main

Comment (TGlide) [reply src/lib/layout/shell.svelte:98]: image
Still happens after merging with main

**Review by ArmanNik — APPROVED** [review]

**Pull request merged**